### ISSUE TITLE
Detect most active GCI repo for each org

### ISSUE THREAD

**Issue by jayvdb** [coala/gci-leaders#165]
Like https://github.com/coala/gci-leaders/issues/164 ...

Im not sure how to get it using open data methodology , but lets try... ;)

**Comment by li-boxuan** [comment]
A possible approach:

Fetch all GCI tasks
Fetch task description, one by one
Use regex to match GitHub/GitLab repo in the task description

API needed is on https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf

**Comment by andrewda** [comment]
I'm not positive on this one, but I don't _think_ you're able to get task descriptions from other orgs using that API? So that would work for getting _our_ most active GCI repo, but not the most active repo for other orgs.

**Comment by li-boxuan** [comment]
> I'm not positive on this one, but I don't think you're able to get task descriptions from other orgs using that API?

Oh really, I didn't know that. People can see tasks from any org on gci website, so I assumed we could get task descriptions using API.

**Comment by andrewda** [comment]
That's true, maybe that's the case. The API just it's very well documented and it didn't look like they gave you the `organization_name` field on most of the requests...

**Comment by jayvdb** [comment]
Ya the API only gives data about one org, so not useful.

There are some hacks around to log in as a student, using oauth, and then you can see all tasks from all orgs.

But all tasks from all orgs are are available to all mentors from any org, but only if you know the URL.

**Comment by jayvdb** [comment]
As a first attempt at this, we could just sort the repos in each github org.

The scraper can not access github dashboards for orgs that it is not a member of.

We could try to get waffle stats out of orgs which have enabled that, e.g. https://waffle.io/coala/coala-bears/metrics/throughput

Ahhhh ... https://github.com/coala/gci-leaders/issues/166 will give us a window into this information.
